modules/claims_api/spec/lib/claims_api/v2/disability_compensation_pdf_mapper_spec.rb

```diff
@@ -13,36 +13,36 @@
         }
       }
     end
-
-    let(:auto_claim) do
-      JSON.parse(
-        Rails.root.join(
-          'modules',
-          'claims_api',
-          'spec',
-          'fixtures',
-          'v2',
-          'veterans',
-          'disability_compensation',
-          'form_526_json_api.json'
-        ).read
-      )
+    let(:active_duty_end_date) { 2.days.from_now.strftime('%Y-%m-%d') }
+    let(:form_data) do
+      temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                             'disability_compensation', 'form_526_json_api.json').read
+      temp = JSON.parse(temp)
+      attributes = temp['data']['attributes']
+      attributes['serviceInformation']['federalActivation'] =
+        {
+          'activationDate' => activation_date,
+          'anticipatedSeparationDate' => anticipated_separation_date
+        }
+      attributes['serviceInformation']['placeOfLastOrAnticipatedSeparation'] = 'Aberdeen Proving Ground'
+      temp['data']['attributes'] = attributes
+      temp
```

Comment:
Same question as above: do we still expect these attributes if we are removing them from the docs?